specs/LaunchingRegisteredProtocols.md

```diff
@@ -0,0 +1,318 @@
+# Background
+
+We are exposing an event that will be raised when an attempt to launch a protocol that is registered with the OS (external protocol) is made.
+When navigating to a URI, the URI scheme determines how to handle the URI.
+Some schemes like http, and https, are resolved by WebView2 and the navigation is handled in the WebView2.
+Other URI schemes may be registered externally to the WebView2 with the OS by other applications.
+Such schemes are external protocols and are handled by launching the registered application with the URI.
+
+The host will be given the option to cancel the external protocol launch with the `LaunchingExternalProtocol` event.
+Cancelling the launch gives the host the opportunity to hide the default dialog, display a custom dialog, and then launch the external protocol themselves.
+
+# Description
+
+This event will be raised before the external protocol launch occurs.
+When an attempt to launch an external protocol is made, the default dialog is displayed in which the user can select `Open` or `Cancel` if the host does not cancel the event.  

```

Comment:
```suggestion
When an attempt to launch an external protocol is made and the host does not cancel the event, the default dialog is displayed in which the user can select `Open` or `Cancel`.  

```
Wordsmithing. Combine the two conditionals up front to make the sentence clearer.

Reply:
Please fix.